scripts/generators/create-package/index.ts

```diff
@@ -7,7 +7,8 @@ import _ from 'lodash';
 import chalk from 'chalk';
 import { spawnSync } from 'child_process';
 import { WorkspaceJsonConfiguration } from '@nrwl/devkit';
-import { findGitRoot, PackageJson } from '../monorepo/index';
+
```

Comment:
fixed type errors and normalized name to follow our nx generarators -> `index.ts` as root